src/db_adapters/transactions.rs

```diff
@@ -25,33 +59,61 @@ pub(crate) async fn store_transactions(
                 chunk
                     .transactions
                     .iter()
-                    .collect::<Vec<&near_indexer::IndexerTransactionWithOutcome>>(),
+                    .enumerate()
```

Comment:
do we need this `enumerate` here? I don't see the use, could you point me where to look if I'm wrong?

Reply:
I had to put it here because we need to pass indexes for collided transactions, they will not be sequential